KEEP/etc/butch-optflags.sh

```diff
@@ -4,14 +4,38 @@
 # this file is sourced by the default butch build template
 
 isgcc3() {
-	local mycc="$CC"
-	[ -z "$mycc" ] && mycc=gcc
-	$mycc --version | grep "3.4.6" >/dev/null
+  local mycc="$CC"
+  [ "$STAGE" = "0" ] && \
+    [ -x "$butch_root_dir/bin/musl-gcc" ] && \
+    mycc="$butch_root_dir/bin/musl-gcc"
+  [ -z "$mycc" ] && mycc=gcc
+  $mycc --version | grep "3.4.6" >/dev/null
 }
 
-optcflags="-fdata-sections -ffunction-sections -Os -g0 -fno-unwind-tables -fno-asynchronous-unwind-tables -Wa,--noexecstack"
+optcflags="-mtune=generic -fdata-sections -ffunction-sections -Os -g0 -fno-unwind-tables -fno-asynchronous-unwind-tables -Wa,--noexecstack -pipe"
```

Comment:
it should unless it's a default option. I think otherwise it uses shm so it's not that big of a deal.